docs/how_to/run_session.rst

```diff
@@ -88,20 +88,17 @@ There is also an interactive timeout value (5 minutes), which is not configurabl
 Close a session
 ---------------
 
-When jobs are all done, it is recommended that you use `session.close()` to close the session. This allows the scheduler to run the next job without waiting for the session timeout,  therefore making it easier for everyone.  You cannot submit jobs to a closed session.
+When the context manager exits or when `session.close()` is explicity called, the session won't accept any more jobs. Any queued jobs will run until completion or until the max time expires.
 
-.. warning::
-  Close a session only after all session jobs **complete**, rather than immediately after they have all been submitted. Session jobs that are not completed will fail.
+When a session that is not accepting jobs has run out of jobs to run, it is immediately closed.
 
```

Comment:
@beckykd should docs updates be in a separate PR? I'll probably need help with the wording on a lot of these - it will also be a bit tricky since once this is released, older versions of `qiskit-ibm-runtime` will behave differently